.drone.yml

```diff
@@ -1,13 +1,54 @@
 ---
 kind: pipeline
+type: docker
 name: fossa
 
 steps:
-- name: fossa
-  image: rancher/drone-fossa:latest
-  settings:
-    api_key:
-      from_secret: FOSSA_API_KEY
-    when:
-      instance:
-        - drone-publish.rancher.io
+  - name: fossa
+    image: rancher/drone-fossa:latest
+    settings:
+      api_key:
+        from_secret: FOSSA_API_KEY
+      when:
+        instance:
+          - drone-publish.rancher.io
+
+---
+kind: pipeline
+type: docker
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: test
+    image: registry.suse.com/bci/golang:1.19
+    commands:
+      - zypper -n install tar gzip
+      - "curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.49.0;"
```

Comment:
The latest release of golangci-lint is [1.51.1](https://github.com/golangci/golangci-lint/releases/tag/v1.51.1). Why not use a newer version?

Reply:
I just pulled this from what Rancher was using I will update it.

Reply:
We don't need to do it now but, we should move this to a GitHub Action (one exists). Then we can have dependabot manage the dependency to keep that up to date.

Reply:
You can see what Fleet is doing with this [here](https://github.com/rancher/fleet/blob/master/.github/workflows/golangci-lint.yml) and [here](https://github.com/rancher/fleet/blob/e1b6f287fffa745cdc6f40bdb70db83a61507b4d/.github/dependabot.yaml#L31-L37).

Reply:
I haven't spent much time looking into actions. How do you decide what should use actions and what should be validated via drone?

Reply:
Drone MUST be used for builds and for tests that go across multiple architectures. s390x isn't something that GitHub Actions can do.

I think other things are just fine in GitHub Actions. Linting is something that doesn't matter per os/arch, for example.